docs/content/cli-commands/npm-install.md

```diff
@@ -113,6 +114,19 @@ after packing it up into a tarball (b).
 
           npm install sax
 
+    * `npm install <alias>@npm:<name>`:
```

Comment:
with `--save`, would this install using the alias notation into package.json?

Reply:
Yes, for example doing `npm install jquery1@npm:jquery@1` will look like this on package.json:
```
  "dependencies": {
    "jquery1": "npm:jquery@^1.12.4"
  }
```

You can also use the alias for npm related tasks, ex: `npm uninstall jquery1`.

Reply:
Great! Should the docs be explicit about that?